### DISCUSSION TITLE
Add feedback URL to the frontend

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#4] [New Feature Requests]
Hey

Thanks for adding feature requests so quickly!

There's one thing missing from the "Feature Requests" though: it's not easily accessible through the website

![screenshot-2023-06-04-12-10-28](https://github.com/solodit/feedback/assets/3029017/9bcd6a4d-2c49-4df1-80d5-28fe28e8e59d)

Maybe adding a GitHub icon next to the Twitter icon on the footer?

**Comment by aviggiano** [comment]
Sorry, I just saw it lol

**Reply by hans-cyfrin** [reply]
We will add a GitHub icon as well. Thank you for a good suggestion.
